harvester/harvester-installer
# Issue 5178

**Problem:**
Please refer [harvester/harvester#5178](https://github.com/harvester/harvester/issues/5178). The additional mgmt-br.VLAN_ID interface would cause confusion and not helpful when debugging packet flow on network problem. 

**Solution:**
Do not create the additional bridge interface while applying network settings

**Test plan:**
1. install harvester
2. input vlan id for the management interface in network settings
3. Verify correct IP address is assigned for the specified VLAN
4. Press Esc to back to step 2 and input different VLAN ID
